outputs.tf

```
@@ -1,14 +1,149 @@
+output "assessment_private_zone" {
+  value       = aws_route53_zone.assessment_private
+  description = "The private DNS zone for this assessment."
+}
+
+output "assessor_portal_instance_profile" {
+  value       = aws_iam_instance_profile.assessorportal
+  description = "The instance profile for the Assessor Portal instances."
+}
+
+output "assessorportal_security_group" {
+  value       = aws_security_group.assessorportal
+  description = "The security group for the Assessor Portal instances."
+}
+
+output "aws_region" {
+  value       = var.aws_region
+  description = "The AWS region where this assessment environment lives."
+}
+
+output "certificate_bucket_name" {
+  value       = var.cert_bucket_name
+  description = "The name of the S3 bucket where certificate information is stored for this assessment."
+}
+
+output "cloudwatch_and_ssm_agent_security_group" {
+  value       = aws_security_group.cloudwatch_and_ssm_agent
+  description = "A security group for _all_ instances.  Allows access to the VPC endpoint resources necessary for the AWS CloudWatch agent and the AWS SSM agent."
+}
+
+output "debian_desktop_instance_profile" {
+  value       = aws_iam_instance_profile.debiandesktop
+  description = "The instance profile for the Debian Desktop instances."
```

I think we have standardized on the capitalization below in this this repo:

```suggestion
  description = "The instance profile for the Debian desktop instances."
```